string object_id
---
float32[] position